sdk/ai/Azure.AI.Agents.Persistent/tests/Samples/Sample_PersistentAgents_Telemetry.cs

```diff
@@ -0,0 +1,373 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+using OpenTelemetry;
+using Azure.Monitor.OpenTelemetry.Exporter;
+
+namespace Azure.AI.Agents.Persistent.Tests;
+
+public partial class Sample_PersistentAgents_Telemetry : SamplesBase<AIAgentsTestEnvironment>
+{
+    [Test]
+    [AsyncOnly]
+    public async Task TracingToConsoleExample()
+    {
+#if SNIPPET
+        var projectEndpoint = System.Environment.GetEnvironmentVariable("PROJECT_ENDPOINT");
+        var modelDeploymentName = System.Environment.GetEnvironmentVariable("MODEL_DEPLOYMENT_NAME");
+#else
+        var projectEndpoint = TestEnvironment.PROJECT_ENDPOINT;
+        var modelDeploymentName = TestEnvironment.MODELDEPLOYMENTNAME;
+#endif
+        #region Snippet:AgentsTelemetrySetupTracingToConsoleAsync
+        var tracerProvider = Sdk.CreateTracerProviderBuilder()
+                        .AddSource("Azure.AI.Agents.Persistent.PersistentAgentsClient") // Add the required sources name
```

Comment:
nit, we usually tell people to enable multiple sources, e.g.

```suggestion
                        .AddSource("Azure.AI.Agents.Persistent.*") // Add the required sources name
```

Reply:
fixed